src/components/TxRow.js

```diff
@@ -1,39 +0,0 @@
-/**
```

Comment:
_unrelated to line_

question(non-blocking): Were this and `WaitVersion.js` removed because they are not used anymore?

Reply:
That's correct :+1: